README.md

```diff
@@ -332,6 +332,8 @@ The vSphere module is only available for Puppet Enterprise 3.7 and later. This m
 
 ## Development
 
+We are experimenting with a new tool for running acceptance tests. It's name is [puppet_litmus](https://github.com/puppetlabs/puppet_litmus) this replaces beaker as the test runner. To run the acceptance tests follow the instructions [here](https://github.com/puppetlabs/puppet_litmus/wiki/Tutorial:-use-Litmus-to-execute-acceptance-tests-with-a-sample-module-(MoTD)#install-the-necessary-gems-for-the-module).
```

Comment:
Is this still an experiment?

Reply:
I'll rephrase this description 
Cheers!